### PR TITLE
Jason updates #2

### PR DESCRIPTION
**Pull request by OPMUSER**

### CODE REVIEW

**Comment by akva2** [comment]
again. don't merge from upstream, rebase on upstream.

**Comment by bska** [comment]
As an aside, the format is called "json" (JavaScript Object Notation) not "jason".

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/share/keywords/000_Eclipse100/R/ROCKCOMP [hunk 1]
```diff
@@ -6,19 +6,35 @@
...
+      "value_type": "STRING",
+      "comment": "This item is ECLIPSE300 specific"
+    },
+    {
+      "item": 5,
+      "name": "CARKZEXP",
+      "value_type": "DOUBLE",
+      "default": 0,
+      "comment": "This item is ECLIPSE300 specific"
+    },
```
Comment (joakim-hove) [inline src/opm/parser/eclipse/share/keywords/000_Eclipse100/R/ROCKCOMP:38]: The final "," after "}" should be removed.

**Comment by joakim-hove** [comment]
1. Make sure to be on the `Jason_Updates` branch
2. `git fetch upstream`
3. `git rebase upstream/master`
4. `git push -f origin Jason_Updates`

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TABDIMS [hunk 2]
```diff
@@ -6,128 +6,152 @@
...
       "name": "NRPVT",
       "value_type": "INT",
       "default": 20
     },
     {
+      "item": 7,
       "name": "MAX_RV_NODES",
       "value_type": "INT"
+      "default": 20,
+      "comment": "This item is ECLIPSE300 specific"
```
Comment (bska) [inline src/opm/parser/eclipse/share/keywords/000_Eclipse100/T/TABDIMS:49]: You need to append a comma to the `"value_type"` line when inserting new elements into this array.  In other words, this needs to say
```json
{
  "item": 7,
  "name": "MAX_RV_NODES",
  "value_type": "INT",
  "default": 20,
  "comment": "This item is ECLIPSE300 specific"
}
```

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**